snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source.cs

```diff
@@ -8,30 +8,31 @@ class WriteTextFiles
     static void Main(string[] args)
     {
 
-        // Example 1: shows how to write synchronously
+        // Example 1: Shows how to write synchronously
```

Comment:
@rpetrusha - In the current live version of the article, there are 4 separate code samples (the 4 snippets from the original sample). They do not have any using statements, class declaration, Main method, so they will not run on their own. All the other samples in this set of articles run on their own, so it seemed confusing not to be able to run these 4 standalone. So there were 2 choices: create 4 separate, runnable samples (which I've now done), or present the entire sample as one and annotate it, which was what I originally did. I agree with you that the 4 samples are more usable. I am removing the numbered list from the topic (replacing it with the original 4 H2s), and most of the introductory code comments from the code, because they repeat the introductory text.